core/src/main/java/org/apache/calcite/util/format/FormatModels.java

```diff
@@ -142,9 +142,7 @@ MI, literalElement(":"), SS, literalElement(" "),
     map.put("%Y", YYYY);
     map.put("%y", YY);
     map.put("%Z", TZR);
-    BIG_QUERY = create(map);
 
-    map.clear();
```

Comment:
why get rid of this

Reply:
That was clearing off all the previous records added to the `FormatElements` map after creating the `BigQuery` FormatModel (line 145), to add new values and create the `POSTGRESQL` model (line 177).

All the below format elements used by `POSTGRESQL` are also supposed to be available in `BigQuery`, so instead of clearing it we can add all those elements and create both models at the end.

Reply:
There would be some restructuring / additions to this set of FormatElements based on the changes required in [CALCITE-6269](https://issues.apache.org/jira/browse/CALCITE-6269)